.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
-dist/
+dist/*
+!_redirects
```

Comment:
Can we specify with absolute path like ‘/dist/_redirects’?

Reply:
unsure, let me try...

Reply:
looks like it!

Reply:
We can add ignored files with force adding ‘-f’ option